recipes/libsigcpp/2.x.x/test_package/CMakeLists.txt

```diff
@@ -1,12 +1,8 @@
 cmake_minimum_required(VERSION 3.6)
 project(test_package)
 
-include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
-conan_basic_setup(TARGET)
-
-find_package(PkgConfig REQUIRED)
-pkg_search_module(SIGCPP IMPORTED_TARGET REQUIRED sigc++-2.0)
+find_package(libsigcpp REQUIRED)
 
```

Comment:
This doesn't appear to work with the new `PkgConfigDeps` generator - not sure if the recipe is missing something or there's an existing issue.

Reply:
You touched a known bug: https://github.com/conan-io/conan/issues/11962

Basically you need a workaround to use pkg_search_module with new cmake generators: https://github.com/conan-io/conan-center-index/blob/master/recipes/libcap/all/test_package/conanfile.py

Not beautiful, but should work.